WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteFragment.kt

```diff
@@ -559,9 +560,7 @@ class MySiteFragment : Fragment(R.layout.my_site_fragment),
     }
 
     private fun hideRefreshIndicatorIfNeeded() {
-        if (swipeToRefreshHelper.isRefreshing) {
```

Comment:
❓ ... so I understand, the guard was eliminated because we start with true (or siteSelected.hasSite) now? We don't need to protect showing the indicator when it shouldn't be showing, correct?

Reply:
That's what I inferred too. I also noticed that `swipeToRefreshHelper` is enabled only `on resume` and if it is disabled, `swipeToRefreshHelper.isRefreshing` check does not allow the indicator to display, the value being false.  This was the main reason I removed the condition,  let me know if you foresee any issues. 😄

Reply:
👍  Yes, we are thinking the same thing. Thanks!!